ceph/rados/core_workflows.py

```diff
@@ -1848,11 +1848,11 @@ def run_parallel_io(self, pool_name: str, obj_name: str, obj_size: int):
         installer_node = self.ceph_cluster.get_nodes(role="installer")[0]
         try:
             out, rc = installer_node.exec_command(
-                sudo=True, cmd="rpm -qa | grep ceph-base"
+                sudo=True, cmd="rpm -qa | grep ceph-common"
```

Comment:
Any reason we are now shifting from ceph-base to ceph-common package?

Reply:
`ceph-base` package installation would require RHEL 9.3 distro, to eliminate this dependency, I change the package to `ceph-common`